src/testing/native-debugging.md

```diff
@@ -0,0 +1,459 @@
+      - In the **panel**:
+        - The log of the Flutter app in the **Debug console**
+
+{% comment %}
+      ![VS Code window with Flutter app paused](/assets/images/docs/testing/debugging/vscode-ui/screens/vscode-debugger-paused.png){:width="100%"}
+{% endcomment %}
+
+### VS Code Flutter debugger
+
+The Flutter plugin for VS Code adds a number of components
+to the VS Code user interface.
+
+#### Changes to VS Code interface
+
+When launched, the Flutter debugger adds debugging tools to the
+VS Code interface.
+
+The following screenshot and table explain the purpose of each tool.
+
+![VS Code with the Flutter plugin UI additions](/assets/images/docs/testing/debugging/vscode-ui/screens/debugger-parts.png)
+
+<div class="table-wrapper" markdown="1">
+| Highlight Color in Screenshot | Bar, Panel, or Tab  | Contents                                                                          |
+|-------------------------------|---------------------|-----------------------------------------------------------------------------------|
+| **Yellow**                    | Variables           | List of current values of variables in the Flutter app                            |
+|                               | Watch               | List of items you chose to track in the Flutter app                               |
+|                               | Call Stack          | Stack of active subroutines in the Flutter app                                    |
+|                               | Breakpoints         | List of exceptions and set breakpoints that you set                               |
+| **Green**                     | `<Flutter files>`   | Files that you are editing                                                        |
+| **Pink**                      | Widget Inspector    | Hierarchy of widgets in the running Flutter app                                   |
+| **Blue**                      | Layout Explorer     | Visual of how the Flutter placed the widget you selected in the Widget Inspector  |
+|                               | Widget Details Tree | List of properties of the widget selected in the Widget Inspector                 |
+| **Orange**                    | Problems            | List of issues the Flutter linter found in the current Flutter file               |
+|                               | Output              | Response that the Flutter app                                                     |
+|                               | Debug Console       | Logs or error messages that the Flutter app generates while debugging             |
+|                               | Terminal            | System shell prompt contained in VS Code                                          |
+{:.table.table-striped}
+</div>
+
+To change where the panel (in **orange**) appears in VS Code,
+go to **View** > **Appearance** > **Panel Position**.
+
+#### VS Code Flutter debugging toolbar
+
+The toolbar allows you to debug as any debugger.
```

Comment:
I'm not 100% sure what this statement means. Perhaps "using any debugger"?

Reply:
Fixed.